### ISSUE TITLE
'Copy to clipboard' button under 'Source' column on the table is not accessible by keyboard keys: A11y_.NetAspireDashboard_Resources_Grid_Keyboard

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3175]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #Benchmark; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG2.1.1; #Keyboard;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: http://localhost:15888/.
2. TAB to "Resources" tab item and hit select the tab by pressing Enter key.
3. Navigate through the table and check if 'copy to clipboard' button is accessible using by keyboard.

### Actual Results:
'Copy to clipboard' button under 'Source' column on the table is not accessible by keyboard keys.

**Similar issue is also observed in throughout the application for "Copy to clipboard" button.**

### Expected Results:
'Copy to clipboard' button under 'Source' column on the table should be accessible by keyboard keys.

### User Impact:
Keyboard users will not be able to interact with the control and will miss the information regarding it.

### Attachment:
![3175_A11y_ NetAspireDashboard_CopyToClipboardButton_Keyboard](https://github.com/dotnet/aspire/assets/90901519/7f1d1e09-dced-46c0-b0a1-fc05e61025e3)

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:04-22-24;

Verified the bug in environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment
[3175 Fixed.webm](https://github.com/dotnet/aspire/assets/90901519/f968c867-4063-4f24-9795-7015f7683c40)
